src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+public class WinningLotto {
+    private final Lotto winningLotto;
+    private final LottoNumber lottoNumber;
```

Comment:
```suggestion
    private final LottoNumber bonuslottoNumber;
```
보너스 번호임을 명시해주는건 어떨까요?